docs/modules/ROOT/pages/extensions.adoc

```diff
@@ -4,17 +4,59 @@ It's possible to extend RuboCop with custom (third-party) cops and formatters.
 
 == Loading Extensions
 
+Starting with RuboCop 1.72, plugins are provided as a way to load extension cop gems.
+When using gems that support plugins, please update the configuration in `.rubocop.yml`
+from `require` to `plugins`.
+
+=== Loading Plugin Extensions
+
+NOTE: The plugin system was introduced in RuboCop 1.72.
+
+Besides the `--plugin` command line option you can also specify ruby
+files that should be loaded with the optional `plugins` directive in the
+`.rubocop.yml` file:
+
+[source,yaml]
+----
+plugins:
+  - rubocop-performance
+----
+
+Most extension gems with plugin support should work with the example above.
+
+The following is an example of a plugin that is not published as a gem:
+
+[source,yaml]
+----
+plugins:
+  - rubocop-extension
+      require_path: path/to/extension/plugin
+      plugin_class_name: RuboCop::Extension::Plugin
+----
+
+There are other ways to specify this.
```

Comment:
Could you elaborate about the other ways?

Reply:
This was intended to guide users to the official lint_roller documentation:
https://github.com/standardrb/lint_roller?tab=readme-ov-file#using-your-plugin

I have adjusted the text slightly to reflect that intention.